libs/ngx-charts-on-fhir/src/lib/fhir-chart-summary/home-measurement-summary.service.ts

```diff
@@ -27,6 +28,8 @@ export class HomeMeasurementSummaryService implements SummaryService {
 export function getOriginalLabel(dataset: Dataset): string | undefined {
   if (dataset.label && dataset.label.endsWith(HOME_DATASET_LABEL_SUFFIX)) {
     return dataset.label.substring(0, dataset.label.length - HOME_DATASET_LABEL_SUFFIX.length);
+  } else if (dataset.label && dataset.label.endsWith(CLINIC_DATASET_LABEL_SUFFIX)) {
+    return dataset.label.substring(0, dataset.label.length - CLINIC_DATASET_LABEL_SUFFIX.length);
   }
```

Comment:
Can we use `chartsOnFhir.group` for grouping datasets instead of label? 
(We could create a separate PR for this if you prefer)

Reply:
for chartsOnFhir.referenceRangeAnnotationId property and this issue we will create separate PR